test/AutoDiff/stdlib/derivative_customization.swift

```diff
@@ -1,8 +1,6 @@
 // RUN: %target-run-simple-swift
 // REQUIRES: executable_test
 
-// REQUIRES: SR12732
```

Comment:
This issue surfaced with `-Osize`. I ought to add a devoted minimal regression test using `-Osize`.

Reply:
Minimal regression test added.